internal/blockchain/ethereum/quorum/geth_provider.go

```diff
@@ -0,0 +1,409 @@
+	return nil
+}
+
+func (p *GethProvider) GetContracts(filename string, extraArgs []string) ([]string, error) {
+	contracts, err := ethereum.ReadContractJSON(filename)
+	if err != nil {
+		return []string{}, err
+	}
+	contractNames := make([]string, len(contracts.Contracts))
+	i := 0
+	for contractName := range contracts.Contracts {
+		contractNames[i] = contractName
+		i++
+	}
+	return contractNames, err
+}
+
+func (p *GethProvider) DeployContract(filename, contractName, instanceName string, member *types.Organization, extraArgs []string) (*types.ContractDeploymentResult, error) {
+	contracts, err := ethereum.ReadContractJSON(filename)
+	if err != nil {
+		return nil, err
+	}
+	return p.connector.DeployContract(contracts.Contracts[contractName], instanceName, member, extraArgs)
+}
+
+func (p *GethProvider) CreateAccount(args []string) (interface{}, error) {
+	l := log.LoggerFromContext(p.ctx)
+	memberIndex := args[2]
+	memberCount := args[3]
+	gethVolumeName := fmt.Sprintf("%s_geth_%s", p.stack.Name, memberIndex)
+	var directory string
+	stackHasRunBefore, err := p.stack.HasRunBefore()
+	if err != nil {
+		return nil, err
+	}
+	if stackHasRunBefore {
+		directory = p.stack.RuntimeDir
+	} else {
+		directory = p.stack.InitDir
+	}
+
+	prefix := strconv.FormatInt(time.Now().UnixNano(), 10)
+	outputDirectory := filepath.Join(directory, "blockchain", fmt.Sprintf("geth_%s", memberIndex), "keystore")
+	keyPair, walletFilePath, err := ethereum.CreateWalletFile(outputDirectory, prefix, keyPassword)
+	if err != nil {
+		return nil, err
+	}
+
+	// Tessera is an optional add-on to the quorum blockchain node provider
+	var tesseraPubKey, tesseraKeysPath string
+	if p.stack.TesseraEnabled {
+		tesseraVolumeName := fmt.Sprintf("%s_tessera_%s", p.stack.Name, memberIndex)
+		tesseraKeysOutputDirectory := filepath.Join(directory, "tessera", fmt.Sprintf("tessera_%s", memberIndex), "keystore")
+		_, tesseraPubKey, tesseraKeysPath, err = CreateTesseraKeys(p.ctx, tesseraImage, tesseraKeysOutputDirectory, "", "tm", keyPassword)
+		if err != nil {
+			return nil, err
+		}
+		l.Info(fmt.Sprintf("keys generated in %s", tesseraKeysPath))
+		l.Info(fmt.Sprintf("generating tessera docker-entrypoint file for member %s", memberIndex))
+		tesseraEntrypointOutputDirectory := filepath.Join(directory, "tessera", fmt.Sprintf("tessera_%s", memberIndex))
+		if err := CreateTesseraEntrypoint(p.ctx, tesseraEntrypointOutputDirectory, tesseraVolumeName, memberCount, p.stack.Name); err != nil {
+			return nil, err
+		}
+	}
+
+	l.Info(fmt.Sprintf("generating quorum docker-entrypoint file for member %s", memberIndex))
+	quorumEntrypointOutputDirectory := filepath.Join(directory, "blockchain", fmt.Sprintf("geth_%s", memberIndex))
+	if err := CreateQuorumEntrypoint(p.ctx, quorumEntrypointOutputDirectory, gethVolumeName, memberIndex, p.stack.QuorumConsensus.String(), p.stack.Name, int(p.stack.ChainID()), p.stack.TesseraEnabled); err != nil {
+		return nil, err
+	}
```

Comment:
The rename stuff specially when I see this

Reply:
Is this on geth being conflicting with quorum? If so, I renamed all geth references to quorum to make it clearer.